dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -75,18 +98,23 @@ INSTALL()
 {
 	make install
 
-	# remove libtool library files
-	rm $libDir/lib*.la
+	rm -f $libDir/*.la
 
 	fixPkgconfig
 
+	#devel and common packages
+	packageEntries devel \
+		$developDir
+
 	packageEntries common \
 		$dataDir/aclocal/vapigen.m4 \
 		$dataDir/aclocal/vala.m4 \
 		$dataDir/vala/Makefile.vapigen
 
 	# Avoid having an empty data/aclocal/ in vala$secondaryArchSuffix
 	rmdir $dataDir/aclocal
+
+	#rmdir $documentationDir
```

Comment:
Now that we have `$manDir` in the base package, it would not be possible to `rmdir $documentationDir`.
So I'd drop that line completely. (No need to keep it, even commented out.)